src/algorithms/reco/MC2ReconstructedParticle.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Sylvester Joosten, Whitney Armstrong, Wouter Deconinck, Dmitry Romanov
+
+#pragma once
+
+#include <algorithms/algorithm.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <edm4hep/MCParticleCollection.h>
+#include <spdlog/logger.h>
+#include <memory>
+
+namespace eicrecon {
+
+using MC2ReconstructedParticleAlgorithm =
+    algorithms::Algorithm<algorithms::Input<edm4hep::MCParticleCollection>,
+                          algorithms::Output<edm4eic::ReconstructedParticleCollection>>;
+
+/**
+     * Converts edm4hep::MCParticle to edm4eic::ReconstructedParticle
+     */
```

Comment:
```suggestion
  * Converts edm4hep::MCParticle to edm4eic::ReconstructedParticle
  */
```

Reply:
Pretty sure clang-format is opinionated here.